desktop/core/src/desktop/lib/export_csvxls.py

```diff
@@ -60,7 +60,7 @@ def file_reader(fh):
   """Generator that reads a file, chunk-by-chunk."""
   while True:
     chunk = fh.read(DOWNLOAD_CHUNK_SIZE)
-    if chunk == '':
+    if not chunk:
```

Comment:
Is this only for when exporting a csv or xls from the editor results?

Reply:
No, this is for general file download.
Any file of any extension triggers this in py3 environment.